visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx

```diff
@@ -0,0 +1,84 @@
+import { PageContentWide } from '@commercetools-frontend/application-components';
+import { ThemeProvider } from '@commercetools-uikit/design-system';
+import { Suite, NestedPages } from '../../test-utils';
+
+export const routePath = '/page-content-container-wide';
+
+const Box = ({ size = 'm' }: { size?: 's' | 'm' | 'l' }) => {
+  return (
+    <div
+      style={{
+        width: '100%',
+        height: size === 's' ? '200px' : size === 'm' ? '60vh' : '2000px',
+        backgroundColor: 'coral',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+    >
+      <h2>Page content container wide</h2>
+    </div>
+  );
+};
+
+export const Component = () => (
+  <Suite>
+    <ThemeProvider theme="test" />
```

Comment:
I think for VRTs it makes sense to check the new theme, since the default theme just renders a `<div>`.
After the migration, we can remove the `ThemeProvider` from here.